src/main/java/pl/wedding/management/model/employee/EmployeeDto.java

```diff
@@ -0,0 +1,17 @@
+package pl.wedding.management.model.employee;
```

Comment:
Jeśli byśmy tworzyli pakiety takim tokiem myslenia to doszlibyśmy do warstwowej architektury, gdzie mielibyśmy 4 pakiety: model, controller, service, repository.

Zgodnie z wybraną przez nas architekturą modularną zmień ścieżkę tej klasy. powinno być .../employee/model zamiast na odwrót.

Reply:
zmieniłem pakiety, będzie w najnowszym commicie